pkg/check/load/load.go

```diff
@@ -12,35 +12,68 @@ import (
 
 	"github.com/ethersphere/bee/v2/pkg/swarm"
 	"github.com/ethersphere/beekeeper/pkg/bee"
+	"github.com/ethersphere/beekeeper/pkg/bee/api"
 	"github.com/ethersphere/beekeeper/pkg/beekeeper"
 	"github.com/ethersphere/beekeeper/pkg/logging"
 	"github.com/ethersphere/beekeeper/pkg/orchestration"
 	"github.com/ethersphere/beekeeper/pkg/scheduler"
 )
 
+type Options struct {
+	ContentSize             int64
+	RndSeed                 int64
+	PostageTTL              time.Duration
+	PostageDepth            uint64
+	PostageLabel            string
+	TxOnErrWait             time.Duration
+	RxOnErrWait             time.Duration
+	NodesSyncWait           time.Duration
+	Duration                time.Duration
+	UploaderCount           int
+	UploadGroups            []string
+	DownloaderCount         int
+	DownloadGroups          []string
+	MaxCommittedDepth       uint8
+	CommittedDepthCheckWait time.Duration
+	IterationWait           time.Duration
+}
+
+func NewDefaultOptions() Options {
+	return Options{
+		ContentSize:             5000000,
+		RndSeed:                 time.Now().UnixNano(),
+		PostageTTL:              24 * time.Hour,
+		PostageDepth:            24,
+		PostageLabel:            "test-label",
+		TxOnErrWait:             10 * time.Second,
+		RxOnErrWait:             10 * time.Second,
+		NodesSyncWait:           time.Minute,
+		Duration:                12 * time.Hour,
+		MaxCommittedDepth:       2,
+		CommittedDepthCheckWait: 5 * time.Minute,
+		IterationWait:           5 * time.Minute,
+	}
+}
+
 func init() {
 	rand.New(rand.NewSource(time.Now().UnixNano()))
 }
 
-// compile check whether Check implements interface
-var _ beekeeper.Action = (*LoadCheck)(nil)
+var _ beekeeper.Action = (*Check)(nil)
 
-// Check instance
-type LoadCheck struct {
+type Check struct {
 	metrics metrics
 	logger  logging.Logger
 }
 
-// NewCheck returns new check
-func NewLoadCheck(log logging.Logger) beekeeper.Action {
-	return &LoadCheck{
+func NewCheck(log logging.Logger) beekeeper.Action {
+	return &Check{
 		metrics: newMetrics("check_load"),
```

Comment:
If the metrics are completly the same as in smoke, we can expose those in smoke, and initialize them here `smoke.NewMetrics("check_load")`.

Reply:
We do not have repeated code, metrics were the same. Now we have different labels, so it is ok to split them com completely.